app/src/components/contextMenu.ts

```diff
@@ -43,12 +47,21 @@ export function initContextMenu(
           items.push({
             label: 'Open Link in New Tab',
             click: () =>
-              createNewTab(
-                outputOptionsToWindowOptions(options),
-                setupNativefierWindow,
+              // Fire a new window event for a foreground tab
+              (window as BrowserWindow).webContents.emit(
+                // event name
+                'new-window',
+                // event object
+                {
+                  newGuest: undefined,
```

Comment:
A lil' comment might be halpful to sum up that thing is

Reply:
@ronjouch Done!